chicago/templates/council_members.html

```diff
@@ -47,7 +47,7 @@ <h4>Look up your council member</h4>
 
 
 {% block extra_js %}
-  {% cache 86400 js_wrapper 'js' %}
+  {% cache 86400 js_wrapper_compare 'js' %}
```

Comment:
using the same cache wrapper on the council members map and compare page is causing an error